src/NzbDrone.Core/Notifications/Webhook/WebhookProxy.cs

```diff
@@ -30,6 +32,11 @@ public void SendWebhook(WebhookPayload body, WebhookSettings settings)
                 request.Headers.ContentType = "application/json";
                 request.SetContent(body.ToJson());
 
+                if (!String.IsNullOrEmpty(settings.Username) || !String.IsNullOrEmpty(settings.Password))
+                {
+                    var networkCredential = new NetworkCredential(settings.Username, settings.Password);
```

Comment:
This is unnecessary as the username and password are pulled out of the network credential in `SetNetworkCredential`